src/cljs/main/broadfcui/main.cljs

```diff
@@ -112,6 +112,33 @@
                   "http://status.firecloud.org/"]
                  " for more information."])}))
 
+(defn status-alert [title message link]
+  [:div {:style {:borderBottom "1px solid" :borderBottomColor (:line-default style/colors)
+                 :backgroundColor (:exception-state style/colors) :padding "1rem"}}
+   [:div {:style {:color "#fff" :marginBottom "0.5rem" :fontWeight 600}}
+    [icons/ExceptionIcon {:size 18 :paddingRight "0.5rem"}] (if title title "Service Alert")]
```

Comment:
Ah, I see. The way we've been dealing with that is to wrap the icon in its own container.